climada/entity/exposures/test/test_crop_production.py

```diff
@@ -58,17 +58,18 @@ def test_set_to_usd(self):
         self.assertEqual(exp.value.shape, (1092,))
         self.assertEqual(exp.value_unit, 'USD / y')
         self.assertEqual(exp.crop, 'mai')
+        self.assertAlmostEqual(exp.value_tonnes[28], 1998.3634803238633)
         self.assertAlmostEqual(exp.value.max(), 51603897.28533253, places=5)
         self.assertAlmostEqual(exp.value.mean(), 907401.9933073953, places=5)
         self.assertEqual(exp.value.min(), 0.0)
         
     
     def test_set_to_usd_unnecessary(self):
```

Comment:
Did you mean to delete test `test_set_to_usd_unnecessary` ? It looks sub-identical to `test_set_to_usd`